client/.gitignore

```diff
@@ -9,7 +9,7 @@ node_modules
 /coverage
 
 # production
-/build
+# /build
```

Comment:
I wouldn't mess with .gitignore unless you do it in a separate commit/PR. Just to keep things organized. :)